packages/eslint-config-airbnb/rules/best-practices.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
     // disallow else after a return in an if
     'no-else-return': 2,
     // disallow use of labels for anything other then loops and switches
-    'no-empty-label': 2,
+    'no-labels': [2, { 'allowLoop': true, 'allowSwitch': true }],
```

Comment:
we should definitely set both of these to `false` - we don't allow GOTO in any form, including breaking to a label.